src/Me/components/List/List.js

```diff
@@ -9,29 +9,20 @@ const Style = {
   `,
 };
 
-export const List = ({ items, children }) => {
-  const _items = useMemo(
-    () =>
-      items
-        ? items.map(item => (
-            <ListItem {...item} key={`${item.type}${item.value}`} />
-          ))
-        : children,
-    [children, items]
-  );
+const renderItems = item => (
+  <ListItem {...item} key={`${item.type}${item.value}`} />
+);
 
-  return <Style.List>{_items}</Style.List>;
+export const List = ({ items, children }) => {
+  const listItems = items?.map(renderItems) ?? children;
```

Comment:
Actually it can be `||` because if `items` is `undefined` so the outcome of the `map`.

Reply:
The reason I'm using `??` and not `||` is to make it clear that if `items` isn't passed (so `null` or `undefined`) then go for `children`. But unlike `||` if user pass falsy value like empty string or `0` for example then I want the browser to throw an error since this is invalid value and you not suppose to pass `items` with `children` so it makes no sense to fallback on it.

Reply:
Right 👍
We need typescript asap 😉

Van you merge?

Reply:
Typescript would be nice addition indeed 🙏